src/contact.ts

```diff
@@ -1,7 +1,8 @@
 /**
+ *
  *   Wechaty - https://github.com/chatie/wechaty
  *
- *   Copyright 2016-2017 Huan LI <zixia@zixia.net>
+ *   @copyright 2016-2017 Huan LI <zixia@zixia.net>
```

Comment:
It would be better if all files share the same header (copyright info, license, etc)

Reply:
@hczhcz thanks! I'll do that later by an util script written by me at https://github.com/Chatie/wechaty/blob/master/script/update-license.ts, in case you are interested at.